prov/tcp/src/xnet_progress.c

```diff
@@ -908,9 +909,10 @@ static int xnet_handle_read_req(struct xnet_ep *ep)
 
 	resp->iov_cnt = 1 + resp->hdr.base_hdr.rma_iov_cnt;
 	resp->hdr.base_hdr.size = resp->iov[0].iov_len;
+	domain = container_of(ep->util_ep.domain, struct xnet_domain, util_domain);
 	for (i = 0; i < resp->hdr.base_hdr.rma_iov_cnt; i++) {
-		ret = ofi_mr_verify(&ep->util_ep.domain->mr_map, rma_iov[i].len,
-				    (uintptr_t *) &rma_iov[i].addr,
+		ret = ofi_mr_verify(&domain->util_domain.mr_map,
```

Comment:
I think this change was added before we moved everything to use the xnet_domain so I think we can take this out